source/c1_impact_op_biodiversiteit_bii/bii_abund_2016_klassen_100m.tif.aux.xml

```diff
@@ -0,0 +1,28 @@
+<PAMDataset>
```

Comment:
Zijn die extra bestanden strikt noodzakelijk? Of volstaan de `.tif` en `.tfw`

Reply:
tif en tfw volstaan, ik delete de rest